content/20.treatments.md

```diff
@@ -505,6 +504,39 @@ Supplementation of vitamin D and maintaining a healthy diet for optimum vitamin
+
+**Potential Mechanisms.**
+Probiotics and next generation probiotics, which are more akin to pharmacological grade supplements, have been associated with multiple potential beneficials effects for allergies, digestive tract disorders and even metabolic diseases through their anti-inflammatory and immunomodulatory effects [@doi:10.1038/nmicrobiol.2017.57; @doi:10.1093/advances/nmy063]. 
+However, the mechanisms by which probiotics affect these various conditions are diverse, heterogeneous, and strain specific [@doi:10.1093/advances/nmy063]. 
+Some of the beneficial effects of probiotics include reducing inflammation by promoting the expression of anti-inflammatory mediators, inhibiting toll-like receptors (TLR) 2 and 4, direct competition with pathogens, the synthesis of antimicrobial substances or other metabolites, improving intestinal barrier function, and/or favourably altering the gut microbiota and the brain-gut axis [@doi:10.1016/j.earlhumdev.2019.05.010; @doi:10.1093/advances/nmy063; @doi:10.1159/000342079]. 
+However, there is also a bi-directional relationship between the lungs and gut microbiota known as the gut-lung axis [@doi:10.1038/mi.2011.55], whereby gut microbial metabolites and endotoxins may affect the lungs via the circulatory system and the lung microbiota in return may affect the gut [@doi:10.1111/cmi.12966]. 
+Therefore, the gut-lung axis may play role in our future understanding of COVID-19 pathogenesis and become a target for probiotic treatments [@doi:10.1016/j.virusres.2020.198018].
+
+**Current Evidence.**
+Probiotics have been associated with the reduction of risk and duration of viral upper respiratory tract infections [@doi:10.1007/s10096-014-2086-y; 10.1002/14651858.CD006895.pub3; 10.1016/j.ijantimicag.2008.11.005]. 
+Meta-analyses have assessed the efficacy of probiotics in viral respiratory infections and reported moderate reductions in the incidence and duration of infection [@doi:10.1002/14651858.CD006895.pub3; @doi:10.1017/S0007114514000075]. 
+Indeed, randomized controlled trials have shown that administering Bacillus subtilis and Enterococcus faecalis [@doi:10.1007/s00134-016-4303-x] or Lactobacillus rhamnosus GG [@doi:10.1164/rccm.200912-1853OC] or Lactobacillus casei and Bifidobacterium breve [@doi:10.1186/s13054-018-2167-x] via the nasogastric tube to ventilated patients reduced the occurrence of ventilator-associated pneumonia in comparison to the respective control groups in studies of viral infections and sepsis. 
+These findings are supported by a recently published meta-analysis [@doi:10.4187/respcare.07097] and are particularly pertinent as there is a significant risk of ventilator-associated pneumonia in COVID-19 patients [@doi:10.6084/m9.figshare.12340496], which can be a challenge for clinicians to diagnose due to the fact that severe COVID-19 infection present with the symptoms of pneumonia [@doi:10.1186/s13054-020-03013-2]. 
```

Comment:
```suggestion
These findings are supported by a recently published meta-analysis [@doi:10.4187/respcare.07097].
There is a significant risk of ventilator-associated pneumonia in COVID-19 patients [@doi:10.6084/m9.figshare.12340496], but it can be challenging for clinicians to diagnose this infection due to the fact that severe COVID-19 infection present with the symptoms of pneumonia [@doi:10.1186/s13054-020-03013-2].
```
Just tried to simplify this text a little